src/resources/views/base/auth/passwords/email.blade.php

```diff
@@ -15,7 +15,7 @@
                 <div class="tab-content">
                   <div class="tab-pane active" id="tab_1">
                     @if (session('status'))
-                        <div class="alert alert-success">
+                        <div class="alert alert-success mt-3">
```

Comment:
The notification of successful sending an email has a large margin-bottom, which also creates asymmetry. ".mt-3" will center it.